<html>
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Yachie lab</title>
  <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&amp;display=swap" rel="stylesheet">
  <link href="https://fonts.googleapis.com/css2?family=Raleway:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&amp;display=swap" rel="stylesheet">
  <link href="https://fonts.googleapis.com/css2?family=Raleway:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&amp;display=swap" rel="stylesheet">
  <link href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900" rel="stylesheet">
  <link href="https://fonts.googleapis.com/css2?family=M+PLUS+Rounded+1c:wght@100;300;400;500;700;800;900&amp;display=swap" rel="stylesheet">
  <link href="https://fonts.googleapis.com/css2?family=Noto+Sans+SC:wght@100..900&amp;display=swap" rel="stylesheet">
  <link rel="stylesheet" href="./css/common.css">
  <link rel="stylesheet" href="./css/content.css">
  <link rel="stylesheet" href="./css/publications.css">
  <link rel="stylesheet" href="./css/chinese.css">
  <link rel="stylesheet" href="./css/instagram.min.css">
  <script src="./js/common.js" defer=""></script>
  <script src="./js/mobile_menu.js" defer=""></script>
</head>

<body>
<div id="container">
<!-- Header -->
<header id="mobile_header">
  <a href="index.html"><img src="./img/blue_yachielablogo.svg" alt="Yachielab logo"></a>
  <div id="menu-icon">
    <div class="menu-icon__bar"></div>
    <div class="menu-icon__bar"></div>
    <div class="menu-icon__bar"></div>
  </div>
</header>

<header id="normal_header"><div class="All Common" style="display: flex; flex-direction: column; justify-content: flex-start;" id="head">
<div class="All Common" style="display: flex; flex-direction: row; justify-content: flex-start;justify-content: flex-between; width: 100%;" id="head-title">
<h1 class="page_title All Common" id="head-title-h1">Yachie Laboratory at UBC and Osaka University</h1><div class="All Common" style="display: flex; flex-direction: row; justify-content: flex-start;" id="top-menu">
<section class="content All Common" id="">
<p><a href="resources.html#software">Software</a></p>
</section>
<section class="content All Common" id="">
<p><a href="resources.html#downloads">Downloads</a></p>
</section>
<section class="content All Common" id="">
<p><a href="collab.html">Collabs</a></p>
</section>
<section class="content All Common" id="">
<p><a href="news.html">Gallery</a></p>
</section>
<section class="content All Common" id="">
<p><a href="https://www.addgene.org/Nozomu_Yachie/">Plasmids</a></p>
</section>
</div>
</div>
<div class=" " style="display: flex; flex-direction: row; justify-content: flex-start;width: 100%;" id="head-lang-logo">
<section class="content All Common" id="frontlogo" style="width: 75%; margin-top:0; z-index: 2;">
<a href="./index.html"><img src="./img/white_yachielablogo.svg" alt="content_img" style="" class="content_img"></a>
</section>
<section class="content All Common" id="backlogo" style="width: 75%; margin-top:0; position: absolute; z-index: 1;">
<a href="./index.html"><img src="./img/blue_yachielablogo.svg" alt="content_img" style="" class="content_img"></a>
</section>
<div class="All Common" style="display: flex; flex-direction: row; justify-content: flex-start;font-family: raleway; justify-content: flex-end; width: 25%;" id="languages">
<section class="content All Common" id="EN">
<p>English</p>
</section>
<section class="content All Common" id="JA">
<p>日本語</p>
</section>
<section class="content All Common" id="ZH">
<p>中文</p>
</section>
</div>
</div>
</div>
</header>

<main>
  <!-- Sidebar Content -->
  <aside><div class="All Common" style="display: flex; flex-direction: column; justify-content: flex-start;" id="sidebar">
<section class="content All English" id="">
<p><a href="index.html">Top</a></p>
</section>
<section class="content All Japanese" id="">
<p><a href="index.html">トップ</a></p>
</section>
<section class="content All Chinese" id="">
<p><a href="index.html">首页</a></p>
</section>
<section class="content All English" id="">
<p><a href="research.html">Research</a></p>
</section>
<section class="content All Japanese" id="">
<p><a href="research.html">研究テーマ</a></p>
</section>
<section class="content All Chinese" id="">
<p><a href="research.html">研究课题</a></p>
</section>
<section class="content All English" id="">
<p><a href="news.html">News</a></p>
</section>
<section class="content All Japanese" id="">
<p><a href="news.html">ニュース</a></p>
</section>
<section class="content All Chinese" id="">
<p><a href="news.html">新闻</a></p>
</section>
<section class="content All English" id="pplV_EN">
<p><a href="people.html">People in Vancouver</a></p>
</section>
<section class="content All Japanese" id="pplV_JA">
<p><a href="people.html">バンクーバーのメンバー</a></p>
</section>
<section class="content All Chinese" id="pplV_ZH">
<p><a href="people.html">温哥华团队</a></p>
</section>
<section class="content All English" id="pplO_EN">
<p><a href="people.html">People in Osaka</a></p>
</section>
<section class="content All Japanese" id="pplO_JA">
<p><a href="people.html">大阪のメンバー</a></p>
</section>
<section class="content All Chinese" id="pplO_ZH">
<p><a href="people.html">大阪团队</a></p>
</section>
<section class="content All English" id="">
<p><a href="publications.html">Publications</a></p>
</section>
<section class="content All Japanese" id="">
<p><a href="publications.html">業績</a></p>
</section>
<section class="content All Chinese" id="">
<p><a href="publications.html">科研成果</a></p>
</section>
<section class="content All English" id="">
<p><a href="joinus.html">Join us</a></p>
</section>
<section class="content All Japanese" id="">
<p><a href="joinus.html">メンバー募集</a></p>
</section>
<section class="content All Chinese" id="">
<p><a href="joinus.html">人才募集</a></p>
</section>
<section class="content All English" id="">
<p><a href="resources.html">Resources</a></p>
</section>
<section class="content All Japanese" id="">
<p><a href="resources.html">リソース</a></p>
</section>
<section class="content All Chinese" id="">
<p><a href="resources.html">资源</a></p>
</section>
<section class="content All Japanese" id="">
<p><a href="internexp.html">インターン体験記</a></p>
</section>
<section class="content All Japanese" id="">
<p><a href="jikkenigaku.html">超生物学—次のX</a></p>
</section>
<section class="content All English" id="">
<p><a href="contact.html">Contact</a></p>
</section>
<section class="content All Japanese" id="">
<p><a href="contact.html">連絡先/アクセス</a></p>
</section>
<section class="content All Chinese" id="">
<p><a href="contact.html">联系方式</a></p>
</section>
</div>
</aside>

  <!-- Main Content -->
  <section class="posts"><h1 class="page_title All Common">Our Vision and Research</h1><section class="content All Common" id="">
<div class="content_img_div filter-willow" style="width: clamp(120px, 50%, 480px);float: right; margin-left: 20px; margin-right: 0;"><img src="./img/DNA%20event%20recording.jpg" alt="content_img"></div>
<p>Starting from a fertilized egg, cells proliferate, pass their genomic information to their offspring, and dynamically change their functions to form diverse tissue structures. Throughout development, intracellular and environmental cues trigger patterns of gene expression that govern cell state transitions and produce additional cellular and environmental cues, leading cells to self-organize into functional clusters within spatially distinct areas. How can these processes be investigated? High-resolution molecular snapshots of cells can be obtained using various omics technologies, but these methods require the destruction of the sample, thereby precluding time-course analyses. Live-cell imaging with fluorescent probes can analyze time-course dynamics but this method is limited to analyzing only a small number of molecules or cells. To overcome this common obstacle in biology, our research program is pioneering two major fileds in biology: <strong><span style="color:#cd853f">DNA event recording</span></strong> and <strong><span style="color:#cd853f">retrospective clone isolation</span></strong>. </p>
</section>
<h2 class="page_title All Common">DNA event recording</h2><section class="content All Common" id="">
<p>In the idea of DNA event recording, molecular and cellular events of a multicellular organism are progressively stored in synthetic “DNA tapes,” like a molecular ticker tape (<a href="https://www.science.org/doi/10.1126/science.abo3471?url_ver=Z39.88-2003">Science 2022</a>). Such a system allows for the readout of historical molecular expression profiles of many cells using high-throughput single-cell sequencing. Analogous to a video camera system, the requirements of DNA event recording systems are: (1) high-capacity DNA “memory” modules embedded in chromosomes; (2) highly sensitive “sensor” modules to capture cell divisions and cell types; (3) information “writer” modules to alter DNA tapes; and (4) high-capacity information “reader” modules to reconstruct complex biological history information written in DNA tapes.</p>
</section>
<section class="paper All Common" id="">
<section class="paper_txt_wo_photo"><p class="author"><span class="member">Masuyama N</span>, <span class="member">Konno N</span> &amp; <span class="member">Yachie N</span></p>
<p class="title">Molecular recorders to track cellular events.</p>
<p class="info"><a class="JT" href="https://www.science.org/doi/10.1126/science.abo3471?url_ver=Z39.88-2003">Science</a> 377, 469-470, 2022 <a class="PMID" href="https://pubmed.ncbi.nlm.nih.gov/35901151/">PubMed</a> <a class="pdf-link" href="./pdf/science.abo3471.pdf">PDF</a></p>
</section>
</section>
<h2 class="page_title All Common">Retrospective Clone Isolation</h2><section class="content All Common" id="">
<div class="content_img_div filter-willow" style="width: clamp(120px, 50%, 480px);float: left; margin-left: 0; margin-right: 20px; "><img src="./img/cloneselect.jpg" alt="content_img"></div>
<p>Another idea to achieve dynamic, high-content observation of progressive systems is retrospective clone isolation. In this concept, cells in a population are first tagged with unique, short DNA barcodes and propagated. A subpopulation is then subjected to a specific assay. After identifying a barcode for a clone of interest, the same clone, or its close relatives, is isolated in a barcode-dependent manner from the initial population or any other subpopulation stored during the experiment. The isolated clone can be subjected to various experiments, including omics measurements and synthetic population reconstitution assays. We recently established a multi-kingdom, highly efficient retrospective clone isolation method using CRISPR base editing (<a href="https://www.biorxiv.org/content/10.1101/2023.01.18.524633v2">bioRxiv 2022</a>). We are currently working on deciphering the molecular priming factors that determine cell fates in different human pluripotent stem cell differentiation and organoid models.</p>
</section>
<section class="paper All Common" id="">
<section class="paper_txt_wo_photo"><p class="author"><span class="member">Ishiguro S</span>,  Ishida K, <span class="member"> Sakata R C</span>, <span class="member"> Mori H</span>,  Takana M,  King S,  Bashth O,  Ichiraku M, <span class="member"> Masuyama N</span>, <span class="member"> Takimoto R</span>, <span class="member"> Kijima Y</span>, <span class="member"> Adel A</span>,  Toyoshima H, <span class="member"> Seki M</span>,  Oh J H,  Archambault A-S,  Nishida K,  Kondo A,  Kuhara S,  Aburatani H,  Klein Geltink R I,  Takashima Y,  Shakiba N &amp; <span class="member"> Yachie N</span></p>
<p class="title">A multi-kingdom genetic barcoding system for precise target clone isolation</p>
<p class="info"><a class="JT" href="https://www.biorxiv.org/content/10.1101/2023.01.18.524633v2">bioRxiv</a> 10.1101/2023.01.18.524633 <a class="pdf-link" href="./pdf/2023.01.18.524633v2.full.pdf">PDF</a></p>
</section>
</section>
<h2 class="page_title All Common">Contribution to Specific Fields</h2><section class="content All Common" id="">
<p>To achieve these visions, we have incrementally worked on diverse topics in biology.</p>
</section>
<h3 class="page_title All Common">Genome editing</h3><section class="content All Common" id="">
<div class="content_img_div filter-willow" style="width: clamp(100px, 25%, 240px);float: right; margin-left: 20px; margin-right: 0;"><img src="./img/genomeediting.jpg" alt="content_img"></div>
<p>In CRISPR-Cas9 genome editing, a guide RNA recruits Cas9 to a target genomic region adjacent to a protospacer adjacent motif (PAM) sequence, 5’-NGG-3’, and Cas9 produces a DNA double-stranded break (DSB). This event promotes either gene deletion or transgene insertion through the induction of different DNA repair pathways, but DSBs are cytotoxic and the DSB-based editing outcomes are unpredictable. In collaboration with Dr. Keiji Nishida, we developed a new genome-editing tool, Target-AID, which fused cytidine deaminase (AID) to a nickase Cas9 and enabled highly precise targeted C→T substitutions without DSBs (<a href="http://science.sciencemag.org/content/353/6305/aaf8729.long">Science 2016</a>). In collaboration with Dr. Osamu Nureki, we also succeeded in increasing the targeting scope of Cas9 from a restrictive NGG to an NG PAM (Cas9-NG) and developed Target-AID-NG (<a href="http://science.sciencemag.org/content/361/6408/1259.long">Science 2018</a>). Furthermore, my team developed a new base editor Target-ACEmax, which enables the simultaneous induction of C→T and A→G substitutions on a target DNA molecule and greatly expands the potential of base editing in therapeutics and biotechnology development (<a href="https://www.nature.com/articles/s41587-020-0509-0">Nature Biotechnology 2020</a>). Furthermore, to explore new genome editing tools apart from the CRIPSR-Cas9 and other characterized genome editing tools, we have developed a tool to rapidly capture periodic and interspaced periodic repeats from genomic and metagenomic resources (<a href="https://academic.oup.com/nar/article/47/2/e8/5124599">Nucleic Acids Research 2019</a>).</p>
</section>
<section class="paper All Common" id="">
<section class="paper_txt_wo_photo"><p class="author">Nishida K, Arazoe T, <span class="member">Yachie N</span>, Banno S, Kakimoto M, Tabata M, Mochizuki M, Miyabe A, Araki M, Hara KY, Shimatani Z &amp; Kondo A</p>
<p class="title">Targeted nucleotide editing using hybrid prokaryotic and vertebrate adaptive immune systems.</p>
<p class="info"><a class="JT" href="http://science.sciencemag.org/content/353/6305/aaf8729.long">Science</a> 353, undefined, 2016 <a class="PMID" href="https://pubmed.ncbi.nlm.nih.gov/27492474/">PubMed</a> <a class="pdf-link" href="./pdf/science.aaf8729.pdf">PDF</a></p>

</section>
</section>
<section class="paper All Common" id="">
<section class="paper_txt_wo_photo"><p class="author">Nishimasu H, Shi X, <span class="member">Ishiguro S</span>, Gao L, Hirano S, Okazaki S, Noda T, Abudayyeh OO, Gootenberg JS, <span class="member">Mori H</span>, Oura S, Holmes B, <span class="member">Tanaka M</span>, <span class="member">Seki M</span>, Hirano H, Aburatani H, Ishitani R, Ikawa M, <span class="member">Yachie N</span>, Zhang F &amp; Nureki O</p>
<p class="title">Engineered CRISPR-Cas9 nuclease with expanded targeting space.</p>
<p class="info"><a class="JT" href="http://science.sciencemag.org/content/361/6408/1259.long">Science</a> 361, 1259-1262, 2018 <a class="PMID" href="https://pubmed.ncbi.nlm.nih.gov/30166441/">PubMed</a> <a class="pdf-link" href="./pdf/science.aas9129.pdf">PDF</a></p>
</section>
</section>
<section class="paper All Common" id="" style="margin-bottom: ;">
<section class="paper_txt_wo_photo"><p class="author"><span class="member">Sakata RC+</span>, <span class="member">Ishiguro S+</span>, <span class="member">Mori H+</span>, <span class="member">Tanaka M</span>, Tatsuno K, Ueda H, Yamamoto S, <span class="member">Seki M</span>, <span class="member">Masuyama N</span>, Nishida K, Nishimasu H, Arakawa K, Kondo A, Nureki O, Tomita M, Aburatani H &amp; <span class="member">Yachie N</span></p>
<p class="title">Base editors for simultaneous introduction of C-to-T and A-to-G mutations.</p>
<p class="info"><a class="JT" href="https://www.nature.com/articles/s41587-020-0509-0">Nature Biotechnology</a> 38, 865-869, 2020 <a class="PMID" href="https://pubmed.ncbi.nlm.nih.gov/32483365/">PubMed</a> <a class="pdf-link" href="./pdf/s41587-020-0509-0.pdf">PDF</a></p>
<p class="info">+Equally contributed</p>

</section>
</section>
<section class="paper All Common" id="">
<section class="paper_txt_wo_photo"><p class="author"><span class="member">Mori H</span>, <span class="member">Evans-Yamamoto D</span>, <span class="member">Ishiguro S</span>, Tomita M &amp; <span class="member">Yachie N</span></p>
<p class="title">Fast and global detection of periodic sequence repeats in large genomic resources.</p>
<p class="info"><a class="JT" href="https://academic.oup.com/nar/article/47/2/e8/5124599">Nucleic Acids Research</a> 47, e8, 2019 <a class="PMID" href="https://pubmed.ncbi.nlm.nih.gov/30304510/">PubMed</a> <a class="pdf-link" href="./pdf/gky890.pdf">PDF</a></p>
</section>
</section>
<h3 class="page_title All Common">Cell lineage tracing</h3><section class="content All Common" id="">
<p>Several methods have been proposed to trace developmental cell lineages of multicellular organisms, whereby chromosome-embedded DNA barcodes are continuously mutated by Cas9 and inherited from mother to daughter cells, whose lineage can be reconstructed according to the mutation patterns at the time of observation. However, none of these technologies has achieved high-resolution lineage tracing. To decipher the map of the whole-body mammalian (mouse) developmental process at the single-cell resolution, we have outlined the key issues and perspectives in this field (<a href="https://www.science.org/doi/10.1126/science.abo3471?url_ver=Z39.88-2003">Science 2022</a>) and is developing new genetic circuits, mouse engineering, and high-performance computing technologies. Target-AID and Target-ACEmax, described above, were developed primarily for the purpose of high-resolution cell lineage tracing. We also developed a new deep distributed computing platform and succeeded in the precise lineage reconstruction of over 235 million mutated sequences generated by a simulator (<a href="https://www.nature.com/articles/s41587-021-01111-2">Nature Biotechnology 2022</a>).</p>
</section>
<section class="paper All Common" id="">
<section class="paper_txt_wo_photo"><p class="author"><span class="member">Masuyama N</span>, <span class="member">Konno N</span> &amp; <span class="member">Yachie N</span></p>
<p class="title">Molecular recorders to track cellular events.</p>
<p class="info"><a class="JT" href="https://www.science.org/doi/10.1126/science.abo3471?url_ver=Z39.88-2003">Science</a> 377, 469-470, 2022 <a class="PMID" href="https://pubmed.ncbi.nlm.nih.gov/35901151/">PubMed</a> <a class="pdf-link" href="./pdf/science.abo3471.pdf">PDF</a></p>
</section>
</section>
<section class="paper All Common" id="">
<section class="paper_txt_wo_photo"><p class="author"><span class="member">Konno N</span>, <span class="member">Kijima Y</span>, <span class="member">Watano K</span>, <span class="member">Ishiguro S</span>, Ono K, <span class="member">Tanaka M</span>, <span class="member">Mori H</span>, <span class="member">Masuyama N</span>, Pratt D, Ideker T, Iwasaki W &amp; <span class="member">Yachie N</span></p>
<p class="title">Deep distributed computing to reconstruct extremely large lineage trees.</p>
<p class="info"><a class="JT" href="https://www.nature.com/articles/s41587-021-01111-2">Nature Biotechnology</a> 40, 566-575, 2022 <a class="PMID" href="https://pubmed.ncbi.nlm.nih.gov/34992246/">PubMed</a> <a class="pdf-link" href="./pdf/s41587-021-01111-2.pdf">PDF</a></p>
</section>
</section>
<h3 class="page_title All Common">Retrospective clone isolation</h3><section class="content All Common" id="">
<p>Questions such as "Did the chemotherapy-resistant clones exist in the initial cell population with a unique cellular status from the beginning?" or "Did any molecular factor underlie the observed stem cell differentiation fates?" highlight many unresolved biological questions. These could be addressed if clones that have once demonstrated a specific phenotype in a later phase of cell progression can be isolated from the initial cell population. The new concept of “retrospective clone isolation” has recently emerged to tackle the above-raised questions. A barcoded cell population is first propagated in such a system, and its subpopulation is subjected to a given assay. After identifying a barcoded clone of interest, the same clone (or its close relative) is isolated in a barcode-specific manner from the initial or any other subpopulation stored during the experiment. The isolated live clone can then be subjected to any following experiments, including omics measurements and reconstitution of a synthetic cell population with the isolates. We have recently established a high-performance retrospective isolation technology CloneSelect using CRISPR base editing (<a href="https://www.biorxiv.org/content/10.1101/2023.01.18.524633v2">bioRxiv 2022</a>). We have demonstrated that CloneSelect is applicable in human cancer cell lines, human pluripotent stem cells, mouse stem cells, yeast cells, and E. coli cells.</p>
</section>
<section class="paper All Common" id="">
<section class="paper_txt_wo_photo"><p class="author"><span class="member">Ishiguro S</span>,  Ishida K, <span class="member"> Sakata R C</span>, <span class="member"> Mori H</span>,  Takana M,  King S,  Bashth O,  Ichiraku M, <span class="member"> Masuyama N</span>, <span class="member"> Takimoto R</span>, <span class="member"> Kijima Y</span>, <span class="member"> Adel A</span>,  Toyoshima H, <span class="member"> Seki M</span>,  Oh J H,  Archambault A-S,  Nishida K,  Kondo A,  Kuhara S,  Aburatani H,  Klein Geltink R I,  Takashima Y,  Shakiba N &amp; <span class="member"> Yachie N</span></p>
<p class="title">A multi-kingdom genetic barcoding system for precise target clone isolation</p>
<p class="info"><a class="JT" href="https://www.biorxiv.org/content/10.1101/2023.01.18.524633v2">bioRxiv</a> 10.1101/2023.01.18.524633 <a class="pdf-link" href="./pdf/2023.01.18.524633v2.full.pdf">PDF</a></p>
</section>
</section>
<h3 class="page_title All Common">Automation of Laboratory and Data Analysis</h3><section class="content All Common" id="">
<p>We have also been contributing to the development of laboratory automation and data analysis automation platforms. We have previously established humanoid robots to automate various molecular and cellular biology experiments and proposed the vision of robotic cloud biology (<a href="http://www.nature.com/nbt/journal/v35/n4/full/nbt.3758.html">Nature Biotechnology 2017</a>). More recently, we have established a new semantic framework to efficiently share reproducible DNA materials and construction protocols within the broad life sciences community (<a href="https://www.nature.com/articles/s41467-022-30588-x">Nature Communications 2022</a>). Furthermore, we have developed a universal tool to interpret and translate high-throughput sequencing read structures, thereby accelerating the development of new sequencing-based assays (<a href="https://www.science.org/doi/full/10.1126/sciadv.add2793?rfr_dat=cr_pub++0pubmed&amp;url_ver=Z39.88-2003&amp;rfr_id=ori%3Arid%3Acrossref.org">Science Advances 2023</a>). In the genomics space, we have established a new theoretical framework to scalably measure the spatial genomics of large tissues without the need for a microscopy system (<a href="https://linkinghub.elsevier.com/retrieve/pii/S2405-4712(23)00243-0">Cell Systems 2023</a>).</p>
</section>
<section class="paper All Common" id="">
<section class="paper_txt_wo_photo"><p class="author"><span class="member">Yachie N</span> &amp; Natsume T</p>
<p class="title">Robotic crowd biology with Maholo LabDroids.</p>
<p class="info"><a class="JT" href="http://www.nature.com/nbt/journal/v35/n4/full/nbt.3758.html">Nature Biotechnology</a> 35, 310-312, 2017 <a class="PMID" href="https://pubmed.ncbi.nlm.nih.gov/28398329/">PubMed</a> <a class="pdf-link" href="./pdf/Yachie2017.pdf">PDF</a></p>
</section>
</section>
<section class="paper All Common" id="">
<section class="paper_txt_wo_photo"><p class="author"><span class="member">Mori H</span> &amp; <span class="member">Yachie N</span></p>
<p class="title">A framework to efficiently describe and share reproducible DNA materials and construction protocols.</p>
<p class="info"><a class="JT" href="https://www.nature.com/articles/s41467-022-30588-x">Nature Communications</a> 13, 2894, 2022 <a class="PMID" href="https://pubmed.ncbi.nlm.nih.gov/35610233/">PubMed</a> <a class="pdf-link" href="./pdf/s41467-022-30588-x.pdf">PDF</a></p>
</section>
</section>
<section class="paper All Common" id="">
<section class="paper_txt_wo_photo"><p class="author"><span class="member">Kijima Y</span>, <span class="member">Evans-Yamamoto D</span>, Toyoshima H &amp; <span class="member">Yachie N</span></p>
<p class="title">A universal sequencing read interpreter.</p>
<p class="info"><a class="JT" href="https://www.science.org/doi/full/10.1126/sciadv.add2793?rfr_dat=cr_pub++0pubmed&amp;url_ver=Z39.88-2003&amp;rfr_id=ori%3Arid%3Acrossref.org">Science Advances</a> 9, eadd2793, 2023 <a class="PMID" href="https://pubmed.ncbi.nlm.nih.gov/36598975/">PubMed</a> <a class="pdf-link" href="./pdf/sciadv.add2793.pdf">PDF</a></p>
</section>
</section>
<section class="paper All Common" id="">
<section class="paper_txt_wo_photo"><p class="author">Greenstreet L, Afanassiev A, <span class="member">Kijima Y</span>, Heitz M, <span class="member">Ishiguro S</span>, King S, <span class="member">Yachie N</span> &amp; Schiebinger G</p>
<p class="title">DNA-GPS: A theoretical framework for optics-free spatial genomics and synthesis of current methods.</p>
<p class="info"><a class="JT" href="https://linkinghub.elsevier.com/retrieve/pii/S2405-4712(23)00243-0">Cell Systems</a> 14, 844-859.e4, 2023 <a class="PMID" href="https://pubmed.ncbi.nlm.nih.gov/37751737/">PubMed</a> <a class="pdf-link" href="./pdf/PIIS2405471223002430.pdf">PDF</a></p>
</section>
</section>
<h2 class="page_title All Common">Positions Available</h2><section class="content All Common" id="">
<p>We are seeking creative and talented postdocs and students.<br>
Please see <strong><a href="./joinus.html">here</a></strong> if you are interested in joining us.</p>
</section>
</section>
  
  <section id="mobile-menu"><div class="All Common" style="display: flex; flex-direction: column; justify-content: flex-start;width: 100%; margin-bottom: 5vw; margin-top: 5vw;" id="mobile-menu-content">
<div class="All Common" style="display: flex; flex-direction: row; justify-content: flex-start;width: 100%; margin-left: 5vw; margin-bottom: 5vw; " id="mobile-lablogo">
<section class="content All Common" id="mobile-ubclogo" style="margin-top: 0; margin-bottom: 0; margin-right:8vw;">
<a href="https://www.ubc.ca/"><img src="./img/navy_ubclogo_h.svg" alt="content_img" style="height: clamp(calc(10px * 44 / 32), calc(7vw * 44 / 32), 44px);margin-top: 0; margin-right: 0; margin-left: 0; margin-bottom: 0; float: left;" class="content_img"></a>
</section>
<section class="content All Common" id="mobile-primelogo" style="margin-top: 0; margin-bottom: 0; margin-right:5vw;">
<a href="https://prime.osaka-u.ac.jp/"><img src="./img/navy_primelogo_h.svg" alt="content_img" style="height: clamp(calc(10px * 44 / 32), calc(7vw * 44 / 32), 44px);margin-top: 0; margin-right: 0; margin-left: 0; margin-bottom: 0; float: left;" class="content_img"></a>
</section>
</div>
<div class="All Common" style="display: flex; flex-direction: row; justify-content: flex-start;width: 100%; margin-left:5vw; align-items: center;" id="mobile-snslogo">
<section class="content All Common" id="mobile-xlogo" style="margin-right: 5vw; margin-bottom: 0; width: auto;">
<a href="https://twitter.com/yachielab"><img src="./img/navy_x.svg" alt="content_img" style="width: clamp(calc(10px * 30 / 32), calc(7vw * 30 / 32), 30px);margin-top: 0; margin-right: 0; margin-left: 0; margin-bottom: 0; float: left;" class="content_img"></a>
</section>
<section class="content All Common" id="mobile-instalogo" style="margin-right: 5vw; margin-bottom: 0; width: auto;">
<a href="https://www.instagram.com/yachielab/"><img src="./img/navy_instagram.svg" alt="content_img" style="width: clamp(10px, 7vw, 32px);margin-top: 0; margin-right: 0; margin-left: 0; margin-bottom: 0; float: left;" class="content_img"></a>
</section>
<section class="content All Common" id="mobile-githublogo" style="margin-right: 5vw; margin-bottom: 0; width: auto;">
<a href="https://github.com/yachielab"><img src="./img/navy_github.svg" alt="content_img" style="width: clamp(calc(10px * 35 / 32), calc(7vw * 35 / 32), 35px);margin-top: 0; margin-right: 0; margin-left: 0; margin-bottom: 0; float: left;" class="content_img"></a>
</section>
<section class="content All Common" id="mobile-addgennelogo" style="margin-right: 5vw; margin-bottom: 0; width: auto;">
<a href="http://www.addgene.org/Nozomu_Yachie/"><img src="./img/navy_addgene.svg" alt="content_img" style="width: clamp(calc(10px * 45 / 32), calc(7vw * 45 / 32), 45px);margin-top: 0; margin-right: 0; margin-left: 0; margin-bottom: 0; float: left;" class="content_img"></a>
</section>
</div>
<div class="All Common" style="display: flex; flex-direction: row; justify-content: flex-start;width: 100%;" id="mobile-links-lang">
<div class="All Common" style="display: flex; flex-direction: column; justify-content: flex-start;width: 70%;" id="mobile-interlinks">
<section class="content All English" id="">
<p><a href="index.html">Top</a></p>
</section>
<section class="content All Japanese" id="">
<p><a href="index.html">トップ</a></p>
</section>
<section class="content All Chinese" id="">
<p><a href="index.html">Top</a></p>
</section>
<section class="content All English" id="">
<p><a href="research.html">Research</a></p>
</section>
<section class="content All Japanese" id="">
<p><a href="research.html">研究テーマ</a></p>
</section>
<section class="content All Japanese" id="">
<p><a href="jikkenigaku.html">超生物学—次のX</a></p>
</section>
<section class="content All Chinese" id="">
<p><a href="research.html">Research</a></p>
</section>
<section class="content All English" id="">
<p><a href="news.html">News</a></p>
</section>
<section class="content All Japanese" id="">
<p><a href="news.html">ニュース</a></p>
</section>
<section class="content All Chinese" id="">
<p><a href="news.html">News</a></p>
</section>
<section class="content All English" id="mpplV_EN">
<p><a href="people.html">People in Vancouver</a></p>
</section>
<section class="content All Japanese" id="mpplV_JA">
<p><a href="people.html">バンクーバーのメンバー</a></p>
</section>
<section class="content All Chinese" id="mpplV_ZH">
<p><a href="people.html">People in Vancouver</a></p>
</section>
<section class="content All English" id="mpplO_EN">
<p><a href="people.html">People in Osaka</a></p>
</section>
<section class="content All Japanese" id="mpplO_JA">
<p><a href="people.html">大阪のメンバー</a></p>
</section>
<section class="content All Chinese" id="mpplO_ZH">
<p><a href="people.html">People in Osaka</a></p>
</section>
<section class="content All English" id="">
<p><a href="publications.html">Publications</a></p>
</section>
<section class="content All Japanese" id="">
<p><a href="publications.html">業績</a></p>
</section>
<section class="content All Chinese" id="">
<p><a href="publications.html">Publications</a></p>
</section>
<section class="content All English" id="">
<p><a href="joinus.html">Join us</a></p>
</section>
<section class="content All Japanese" id="">
<p><a href="joinus.html">メンバー募集</a></p>
</section>
<section class="content All Japanese" id="">
<p><a href="internexp.html">インターン体験記</a></p>
</section>
<section class="content All Chinese" id="">
<p><a href="joinus.html">Join us</a></p>
</section>
<section class="content All English" id="">
<p><a href="resources.html#software">Software</a></p>
</section>
<section class="content All Japanese" id="">
<p><a href="resources.html#software">ソフトウェア</a></p>
</section>
<section class="content All Chinese" id="">
<p><a href="resources.html#software">Software</a></p>
</section>
<section class="content All English" id="">
<p><a href="resources.html#downloads">Downloads</a></p>
</section>
<section class="content All Japanese" id="">
<p><a href="resources.html#downloads">ダウンロード</a></p>
</section>
<section class="content All Chinese" id="">
<p><a href="resources.html#downloads">Downloads</a></p>
</section>
<section class="content All English" id="">
<p><a href="contact.html">Contact</a></p>
</section>
<section class="content All Japanese" id="">
<p><a href="contact.html">連絡先/アクセス</a></p>
</section>
<section class="content All Chinese" id="">
<p><a href="contact.html">Contact</a></p>
</section>
</div>
<div class="All Common" style="display: flex; flex-direction: column; justify-content: flex-start;width: 30%;" id="mobile-lang">
<section class="content All Common" id="mobileEN">
<p>English</p>
</section>
<section class="content All Common" id="mobileJA">
<p>日本語</p>
</section>
<section class="content All Common" id="mobileZH">
<p>中文</p>
</section>
</div>
</div>
</div>
</section>
</main>

<!-- Footer -->
<footer><div class="All Common" style="display: flex; flex-direction: row; justify-content: flex-start;padding-bottom: 8vw;" id="foot">
<div class="All Common" style="display: flex; flex-direction: column; justify-content: flex-start;" id="foot-address">
<section class="content All Common" id="" style="margin-bottom:2em;">
<p><strong>Yachie Laboratory</strong><br>
School of Biomedical Engineering (SBME)<br>
The University of British Columbia (UBC)<br>
<br>
Biomedical Research Centre<br>
2222 Health Sciences Mall<br>
Vancouver, BC Canada V6T 1Z3</p>
</section>
<section class="content All English Chinese" id="">
<p><strong>Yachie Laboratory</strong><br>
Premium Research Institute for Human Metaverse Medicine (PRIMe)<br>
Osaka University<br>
<br>
The Center of Medical Innovation and Translational Research<br>
2-2, Yamadaoka<br>
Suita, Osaka, Japan 565-0871</p>
</section>
<section class="content All Japanese" id="">
<p><strong>谷内江研究室</strong><br>
大阪大学ヒューマン・メタバース疾患研究拠点<br>
<br>
〒565-0871 大阪府吹田市山田丘2-2<br>
最先端医療イノベーションセンター
<br>
<br></p>
</section>
</div>
<div class="All Common" style="display: flex; flex-direction: column; justify-content: flex-start;" id="foot-logo">
<div class="All Common" style="display: flex; flex-direction: row; justify-content: flex-start;height: 25%;" id="foot-lablogo">
<section class="content All Common" id="foot-ubclogo">
<a href="https://www.ubc.ca/"><img src="./img/white_ubclogo_h.svg" alt="content_img" style="margin-top: 0; margin-right: 0; margin-left: 0; margin-bottom: 0; float: right;" class="content_img"></a>
</section>
<section class="content All Common" id="foot-primelogo">
<a href="https://prime.osaka-u.ac.jp/"><img src="./img/white_primelogo_h.svg" alt="content_img" style="margin-top: 0; margin-right: 0; margin-left: 0; margin-bottom: 0; float: right;" class="content_img"></a>
</section>
</div>
<section class="content All Common" id="foot-yachielablogo">
<a href="./index.html"><img src="./img/white_yachielablogo.svg" alt="content_img" style="margin-top: 0; margin-right: 0; margin-left: 0; margin-bottom: 0; float: right;" class="content_img"></a>
</section>
<div class="All Common" style="display: flex; flex-direction: row; justify-content: flex-start;height: 50%;" id="foot-snslogo">
<section class="content All Common" id="foot-xlogo">
<a href="https://twitter.com/yachielab"><img src="./img/white_x.svg" alt="content_img" style="width: clamp(calc(10px * 30 / 32), calc(2.2vw * 30 / 32), 39.6px);margin-top: 0; margin-right: 0; margin-left: 0; margin-bottom: 0; float: right;" class="content_img"></a>
</section>
<section class="content All Common" id="foot-instalogo">
<a href="https://www.instagram.com/yachielab/"><img src="./img/white_instagram.svg" alt="content_img" style="width: clamp(10px, 2.2vw, 42.24px);margin-top: 0; margin-right: 0; margin-left: 0; margin-bottom: 0; float: right;" class="content_img"></a>
</section>
<section class="content All Common" id="foot-githublogo">
<a href="https://github.com/yachielab"><img src="./img/white_github.svg" alt="content_img" style="width: clamp(calc(10px * 35 / 32), calc(2.2vw * 35 / 32), 46.2px);margin-top: 0; margin-right: 0; margin-left: 0; margin-bottom: 0; float: right;" class="content_img"></a>
</section>
<section class="content All Common" id="foot-addgennelogo">
<a href="http://www.addgene.org/Nozomu_Yachie/"><img src="./img/white_addgene.svg" alt="content_img" style="width: clamp(calc(10px * 45 / 32), calc(2.2vw * 45 / 32), 59.4px);margin-top: 0; margin-right: 0; margin-left: 0; margin-bottom: 0; float: right;" class="content_img"></a>
</section>
</div>
</div>
</div>
</footer>
</div>

</body></html>